contrib/pipelines/secadvisor/README.md

```diff
@@ -2,33 +2,223 @@
+    subscriber_password:
+  classify:
+    # cluster_net_masks:
+      # - 10.0.0.0/8
+      # - 172.16.0.0/12
+      # - 192.168.0.0/16
+  filter:
+    # excluded_tags:
+      # - internal
+      # - other
+      # - ingress
+      # - egress
+  transform:
+    sa:
+      # exclude_started_flows: true
+  store:
+    type: s3
+    s3:
+      # -- client parames --
+      endpoint: http://127.0.0.1:9000
+      region: local
+      bucket: bucket
+      access_key: user
+      secret_key: password
+      # api_key: key
+      # iam_endpoint: https://iam.cloud.ibm.com/identity/token
+      object_prefix: logs
+      # -- bulk store params --
+      max_flows_per_object: 6000
+      max_seconds_per_object: 60
+      max_seconds_per_stream: 86400
+      max_flow_array_size: 100000
+```
+
+Entries that begin with '#' are comments and are ignored. Be sure to include/exclude the appropriate configuration parameters.
+
+Each instance of secadvisor should use a distinct host_id or should use "" (the empty host id).
+If a host_id (other than "") is in use by one instance of the security advisor (pipeline), it may not be used by another instance of the security advisor (pipeline).
+
+Set "endpoint" to the object store endpoint URL, e.g. https://s3.us.cloud-object-storage.appdomain.cloud.
+Set the credentials used for authenticating to the object store service.
+You can either use AWS-style HMAC keys by setting "access_key" and "secret_key", or use IBM IAM OAuth by setting "api_key".
+When using IBM IAM Oauth (when setting "api_key"), the default IBM IAM authentication endpoint will be used: https://iam.ng.bluemix.net/oidc/token, if none is specified.
+You can customize this endpoint by setting a different "iam_endpoint" to an alternative URL string.
+
+Set "bucket" to the name of the destination bucket where flows will be dumped.
+If your object store service requires a region specification, set "region" appropriately. This setting is not necessary when using IBM COS.
+
+The subnets specified in cluster_net_masks are used to determine whether a flow is internal, ingress, or egress.
+Enter in the list all of the subnets that you consider to be inside your domain of security.
+
+The types of "excluded_tags" that are recognized are: internal, ingress, egress, other.
+The "internal" flag refers to flows that begin and end inside your domain (as defined by "cluster_net_masks").
+The "ingress" flag refers to flows whose source is ouside your domain and whose target is inside your domain.
+The "egress" flag refers to flows whose source is inside your domain and whose target is outside your domain.
+The "other" flag refers to flows whose source and target are both outside your domain.
+For example, if you want to capture only those flows that go between your domain and outside your domain (ingress and egress), then specify "excluded_tags" to include "internal" and "other".
+
+Be sure to set the parameters max_flow_array_size, etc, to reasonable values.
```

Comment:
what is a reasonable value? I rather that the default is reasonable and thus user does not have to fiddle with this parameter